ansible/playbooks/adhoc/bootstrap-fedora.yml

```diff
@@ -0,0 +1,4 @@
+- hosts: OSEv3
```

Comment:
@joelddiaz let's leave this in openshift-ansible for now and they can move it to contrib if / when they want to.